samplerbox.py

```diff
@@ -251,7 +263,8 @@ def ActuallyLoad():
             for i, pattern in enumerate(definitionfile):
                 try:
                     if r'%%volume' in pattern:        # %%paramaters are global parameters
-                        globalvolume *= 10 ** (float(pattern.split('=')[1].strip()) / 20)
+                        globalsoundvolume *= 10 ** (float(pattern.split('=')[1].strip()) / 20)
+                        globalvolume = globalsoundvolume * (globalvolumeknob/127)
```

Comment:
Why 127? Magic number?

with this change `globalvolume` is now ~ 0.5 of what it was before.

Reply:
There are no "magic numbers". 
As above, MIDI message values are 7 bits long, thus range between 0-127. 
thus: 
0 - no volume
127 - full volume
0/127 = 0
127/127 = 1